lib/router.js

```diff
@@ -35,7 +35,9 @@ var queryString = {
             if (Object.prototype.hasOwnProperty.call(obj, key)) {
                 var value = obj[key];
                 if (Array.isArray(value)) {
-                    value.forEach((v) => params.append(key, v));
+                    value.forEach(function (v) {
```

Comment:
I'm surprised babel wouldn't convert this.

Reply:
It was working perfectly , but only for tests :p